src/backend/metal/src/command.rs

```diff
@@ -1226,6 +1239,57 @@ impl CommandBuffer {
         self.state.stencil_back_reference = back;
         soft::RenderCommand::SetStencilReferenceValues(front, back)
     }
+
+    fn set_stencil_mask_values(
+        &mut self,
+        front_back_read_masks: Option<(pso::StencilValue, pso::StencilValue)>,
```

Comment:
I'm confused about the semantics of this `Option`. Does it mean that the read masks are not used, or that they are not provided (but could still be used)?

Reply:
Just not provided – the current callsites are basically providing either read or write masks. Originally I had two functions `set_stencil_read_mask_values` and `set_stencil_write_mask_values` but I wanted to collapse them together to reduce the boilerplate. Is there a way to make this cleaner?